client/hwlib/debian/tests/control

```diff
@@ -0,0 +1,4 @@
+Test-Command: /usr/share/cargo/bin/cargo-auto-test hwlib 0.0.1-ppa1 --all-targets --all-features
+Features: test-name=rust-hwlib:@
+Depends: dh-cargo (>= 23), cargo, rustc, libssl-dev, pkg-config, @
```

Comment:
`pkg-config` or `pkgconf`, I think the latter is more nimble and you use it everywhere else?

Reply:
I've updated it in the latest changes with `Depends: @builddeps@`